recipes-thunder-lgi/thunder-4.3.0/0132-HUMAXEOSR-995-Block-plugins-activation-when-WPE.patch

```diff
@@ -35,31 +32,22 @@ index 8691697e..f0700833 100644
  
                      if( _administrator.Configuration().LegacyInitialize() == false ) {
                          Deactivate(reason::INITIALIZATION_FAILED);
-@@ -961,6 +965,7 @@ POP_WARNING()
- 
-     void Server::Close()
-     {
-+        _services.BlockPluginActivation();
```

Comment:
is _services.BlockPluginActivation() still called somewhere? (the invocation seems to be removed from the patch)